packages/lex-cli/src/util.ts

```diff
@@ -145,6 +110,26 @@ export function applyFileDiff(diff: FileDiff[]) {
   }
 }
 
+function printZodError(node: ZodFormattedError<any>, path = ''): boolean {
+  if (node._errors?.length) {
+    console.log(chalk.red(`Issues at ${path}:`))
+    for (const err of dedup(node._errors)) {
+      console.log(chalk.red(` - ${err}`))
+    }
+    return true
+  } else {
+    for (const k in node) {
+      if (k === '_errors') {
+        continue
+      }
+      if (printZodError(node[k], `${path}/${k}`)) {
+        // return true
+      }
```

Comment:
Choose your own adventure!  (Or ignore lol—I wasn't totally sure what the intent was here.)
```suggestion
      if (printZodError(node[k], `${path}/${k}`)) {
        return true
      }
```

```suggestion
      printZodError(node[k], `${path}/${k}`)
```

Reply:
Haha nice, yeah obviously I forgot to cleanup after deciding how to do that algorithm. What I settled on is _okay_ but could be much better. If you ever get frustrated with the output of lex-cli for errors in lexicons, you know where to look.